automator/automator.go

```diff
@@ -41,7 +41,7 @@ func (a *Automator) Start(errorLogger log.Logger) {
 		for _, inst := range insts {
 			for service, conf := range inst.Config.Services {
 				if conf.Policy() == flux.PolicyAutomated {
-					a.cfg.Jobs.PutJob(inst.ID, jobs.Job{
+					a.cfg.Jobs.PutJobIfNotExists(inst.ID, jobs.Job{
```

Comment:
I expected these to be the other way around -- that is, `PutJob` checks if the job already exists as it does now, and a variant that doesn't do the duplicate check for use if you're already in a job and you want to reschedule it -- simply because that would keep most invocations as they were.